packages/crawler/src/authenticator/authenticator.ts

```diff
@@ -0,0 +1,67 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import Apify from 'apify';
+import * as Puppeteer from 'puppeteer';
+import { AuthenticationStep, AuthenticationFlow } from './authentication-flow';
+
+export class Authenticator {
+    public constructor(
+        protected authenticationFlow: AuthenticationFlow,
+        private readonly logger: typeof Apify.utils.log = Apify.utils.log,
```

Comment:
we can use console.log() as we do across crawler package. Do we have any benefits from Apify logger?

Reply:
this was so we could mock and test our log output.  If this is unnecessary I can use console.log

Reply:
console.log() probably will have less dependency footprint.